src/resources/views/crud/fields/image.blade.php

```diff
@@ -1,13 +1,23 @@
 @php
-    $prefix = isset($field['prefix']) ? $field['prefix'] : '';
+    $prefix = $field['prefix'] ?? '';
```

Comment:
About this, and since we are going to refactor, should we keep using `$field['prefix'] = $field['prefix'] ?? '';` and use $field['prefix'] across the whole file ? same for `$disk`. 

I think it's consistent with what we have in the other fields.

Let me know.

Reply:
I agree 👌
Do you think we should do the same for `$disk = $field['disk'] ?? null;` ?
I think it makes sense, let me know.

Reply:
I've just commited the changes, thank you for the suggestion @pxpm 👌